X2WOTCCommunityPromotionScreen/Src/X2WOTCCommunityPromotionScreen/Classes/X2DownloadableContentInfo_X2WOTCCommunityPromotionScreen.uc

```diff
@@ -73,6 +76,19 @@ static final function Neuter_NPSBD_UISL()
 }
 // End Issue #26
 
+// Start Issue #85
+static final function Neuter_VariousOptions_UISL()
+{
+	local UIScreenListener CDO;
+
+    CDO = UIScreenListener(class'XComEngine'.static.GetClassDefaultObject(class'VariousOptions.AdvancedAbilityButton'));
```

Comment:
What happens at compile time with this import if the compiler doesn't see this class, and what happens at runtime if this package doesn't exist? I'd feel more comfortable if this avoided an import and instead used `FindObject` or something to find this CDO.

Reply:
Oh damn, you're right, this is supposed to be get CDO by name. Will adjust.